packages/components/src/styles/components/app-header.scss

```diff
@@ -76,6 +75,30 @@
   .hds-dropdown-toggle-button,
   .hds-dropdown-toggle-icon {
     @include hds-interactive-dark-theme();
+
+    // disabled state:
+    &:disabled,
+    &[disabled],
+    &.mock-disabled,
+    &:disabled:focus,
+    &[disabled]:focus,
+    &.mock-disabled:focus,
+    &:disabled:hover,
+    &[disabled]:hover,
+    &.mock-disabled:hover {
+      @include hds-interactive-dark-theme-state-disabled();
```

Comment:
[todo] I think the `@use "../mixins/interactive-dark-theme" as *;` should be added to this file.

Reply:
> [todo] I think the `@use "../mixins/interactive-dark-theme" as *;` should be added to this file.

It's there above. I would have gotten an error with the `@include`s if it had been missing.

Reply:
I updated the Showcase to include disabled button and HomeLink examples.